src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -117,13 +117,13 @@ export default class WaypointModel {
     }
 
     /**
-     * Return the x/y position array from `#_position` property.
+     * Return the waypoint's `StaticPositionModel`
      *
      * @property position
      * @return {array<number>}
      */
     get position() {
-        return this._position.position;
+        return this._position;
```

Comment:
this is a big change.  do you mean to return the actual _instance_ here?  if you do, please be sure to update the doc block with the correct data type.